doc/articles/get-started.md

```diff
@@ -43,13 +43,19 @@ If XAML Intellisense isn't working on a freshly-created project, try the followi
 3. Close and reopen Visual Studio.
 4. Reopen XAML documents.
 
+### Creating an app with Uno - tutorial
+
+See the [Creating an App tutorial](getting-started-tutorial-1.md) for a step-by-step guide to creating a working app with Uno.
+
+![tutorial-screenshot](Assets/quick-start/tutorial-screenshot.png)
+
 ## Next steps
 
 * The Uno Platform promotes a 'UWP-first' development approach. Microsoft's [documentation](https://docs.microsoft.com/en-us/windows/uwp/develop/) on the UWP framework is an essential resource.
 
 * For Uno-specific information, consult the docs here, including a list of [supported features](supported-features.md) and [general development tips](using-uno-ui.md).
 
-* If you're not sure how to do something with Uno, and you think others could benefit from the knowledge, you can post the question on [StackOverflow](https://stackoverflow.com/questions/ask?tags=uno-platform) using the uno-platform tag. Remember to check if an answer [already exists](https://stackoverflow.com/questions/tagged/uno-platform) and follow StackOverflow's [question guidelines](https://stackoverflow.com/help/how-to-ask). 
+* If you're not sure how to do something with Uno, and you think others could benefit from the knowledge, you can post the question on [StackOverflow](https://stackoverflow.com/questions/ask?tags=uno-platform) using the 'uno-platform' tag. Remember to check if an answer [already exists](https://stackoverflow.com/questions/tagged/uno-platform) and follow StackOverflow's [question guidelines](https://stackoverflow.com/help/how-to-ask). 
```

Comment:
The first link is sending me to a logging page. According to the link, the behavior looks weird.

Reply:
Oh maybe you can't ask question if you are not logged. Can you confirm this ?

Reply:
Yes, you need to have an account and be logged in to ask a question (not sure since when?). Seems like it doesn't show the link if we try to use the url with a pre-populated tag. I'll add a note that you need an account.